packages/foundation/src/next/composed.tsx

```diff
@@ -145,7 +145,13 @@ export function composed<
       _defaultStyles: displayName ? finalStyles : styles
     } as TViewProps & IDefaultSlotProps<any>;
 
-    return view ? view(viewProps) : null;
+    const Slots = options.slots
+      ? typeof options.slots === 'function'
+        ? getSlots(viewProps, options.slots(viewProps))
+        : getSlots(viewProps, options.slots)
+      : undefined;
```

Comment:
Do you mean to return a default slot for each of the slot keys passed via the interface?

Reply:
Assuming you mean `composed` default rather than component default: Yes, but it's an error condition. It should never be hit in code, and we should throw an error if it does happen. The ultimate parent BaseComponent should define a default slot implementation for every slot.

Since we can't iterate over keys, I was trying to think of ways we can do this. I see you're just defining an empty object, does that blow up? (I was thinking of other ways we can do this, like [proxies](https://stackoverflow.com/questions/4569296/javascript-overloading-dot/4569315#4569315) to override the default key getter, which could throw an error and return null. I haven't used these, though, and not sure if they polyfill.)

In any case, the "default" impl provided by `composed` should never be triggered in production and would definitely be an exception.